### ISSUE TITLE
[Bug]: mongodb module not loading

### ISSUE THREAD

**Issue by fanpero87** [beyondcode/herd-community#285]
### Operating system version

macOS Sonoma 14.2.1

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.3.1(Build:18)

### PHP Version

PHP 8.2.14

### Bug description

I want to install this package on my Laravel app: `composer require mongodb/laravel-mongodb:4.0.0`, but I get an error saying that I need to have the `mongodb` php extension. 
I did the following:
1. Ran `pecl install mongodb`
2. Add this to the php.ini file `mongodb=/opt/homebrew/lib/php/pecl/20230831/mongodb.so`
3. Ran `herd use 8.2`
4. Ran `brew services stop php` and `brew services start php`

When I ran the `php -m` command I do not see mongodb as a module and if I try to run the composer command on my project I still get the error.
I'm not sure what step am I missing so that I can load the module on my Herd config

Thanks.

### Steps to reproduce

Make a brand new Laravel all and run the composer require command.

**Comment by thecrazybob** [comment]
Step 2 should be `extension=/opt/homebrew/lib/php/pecl/20230831/mongodb.so` instead of `mongodb=`. In step 4, use `herd restart` instead of using brew to restart php.

**Comment by fanpero87** [comment]
Thanks. I did that and it dit not help.
However, it seems that it does work using `php8.3`.

This config works for me:
on this path: `~/Library/Application Support/Herd/config/php/83`
I added this line:
`extension=/opt/homebrew/Cellar/php/8.3.1/pecl/20230831/mongodb.so`
Then, restart herd. 
I also select 8.3 instead of 8.2 on the herd menu, and now the `mongodb` module loads.

I also checked this path: `/opt/homebrew/etc/php/` and inside the `php.ini` file of the 8.3 folder I did see `extension="mongodb.so"` as the first line inside the file.

Anyhow, I could not make it work using `php8.2` but at least it does work with `php8.3`
